### ISSUE TITLE
frame-benchmarking support

### ISSUE THREAD

**Issue by clangenb** [open-web3-stack/open-runtime-module-library#998]
I have noticed that the runtime didn't really like when the benchmarking macro from this repository is mixed with the benchmarking macro from substrate. Hence, I have re-implemented the `orml-vesting` benchmarks with substrate's macro, [here](https://github.com/ajuna-network/ajuna-pallets/tree/develop/orml-pallets-benchmarking).

It only contains the benchmarks for the `orml-vesting` pallet currently. Is this something that you'd be interested in having upstreamed?

**Comment by xlc** [comment]
yeah that will be great. the new benchmark syntax and omni bencher have solved a lot of issues that the old one had and now it is a good time to migrate to it
